tests/datasets/base.py

```diff
@@ -33,13 +33,19 @@ def get_subclasses(cls):
         subclasses = []
         for subclass in cls.__subclasses__():
             if ABC not in subclass.__bases__:
-                subclasses.append(subclass)
+                try:
+                    subclass()
```

Comment:
I suspect this is here to catch the classes like the [BooleanGenerator](https://github.com/sdv-dev/RDT/blob/50635411bbb7419cab48a436b036eb9702984c27/tests/datasets/boolean.py#L11) that are intended to be inherited by other classes that implement the `generate` method.

This should be achieved by adding `ABC` to the parent classes, so these steps here are not necessary

```
class BooleanGenerator(ABC, DatasetGenerator):
    ...
```